pages/validator/external-chains/meta.json

```diff
@@ -10,6 +10,5 @@
   "hero": "Heroes of NFT",
   "kava": "Kava",
   "moonbeam": "Moonbeam",
-  "Optimism": "Optimism",
```

Comment:
In local its causing to create  two optimism links in the index, don't know why, waiting for the preview to appear - will add back if that not the case there

Reply:
Preview looks okay so looks like we don't we don't need it

Reply:
Perhaps its because `Optimism` key is capitalized? Try  `"optimism": "Optimism"`, see if that makes any difference.

Reply:
That was it !